src/styles/blocks/card.scss

```
@@ -0,0 +1,112 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 50px;
+}
+
+.card {
+  width: 200px;
+  background-color: $background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__pic {
+    margin: auto;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__name {
+    margin-top: 40px;
+    margin-bottom: 4px;
+    padding: 0 16px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    color: $main-text-color;
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    padding: 0 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $second-text-color;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 12px;
+
+    &--count {
```

```suggestion
    &-count {
```

You made it modificator
Instead, make it just an element of the card

Add margin to the outer container(card__reviews) instead of adding it here